packages/paste-website/src/pages/components/toast/index.mdx

```diff
@@ -0,0 +1,390 @@
+
+Be cautious when using a warning toast because they can be stressful for a user to see. If the information you're trying to convey is not very critical, consider using a neutral toast instead.
+
+<LivePreview scope={{Toast, Text, Anchor}} language="jsx">
+  {`<Toast onDismiss={() => alert('dismissed')} variant="warning">
+  <Text as="span">
+    The phone number has been deleted. Please verify there are no functions tied to that number <Anchor href="#">here</Anchor>.
+  </Text>
+</Toast>`}
+</LivePreview>
+
+### Error Toast
+
+Use an error toast to help a user understand an issue that's preventing them from completing their current action. Provide a way to resolve the issue either in the underlying page content or in the toast.
+
+Be cautious when using an error alert because they can be stressful for a user to see.
+
+<LivePreview scope={{Toast, Text}} language="jsx">
+  {`<Toast onDismiss={() => alert('dismissed')} variant="error">
+  <Text as="span">
+    There was an error when deleting the profile. Please try again.
+  </Text>
+</Toast>`}
+</LivePreview>
+
+### Toast with automatic time-out
+
+Add an automatic time-out to a toast if it doesn't communicate any information that would help a user continue a flow, and would be annoying to the user to have to explicitly acknowledge and close.
+
+In most cases, it's better to dismiss the toast when a user reloads or navigates away from a page, rather than timing it out. Add an automatic time-out to a success toasts if the only thing it does it confirm that the action was completed. If the toast has any additional information or interactive content, like a link or button, don't make the toast time out.
+
+Add an automatic time-out if all these conditions are true:
+
+- The user wouldn't need to remember or copy the information in order to continue with their current flow, or any future flow.
+- The toast doesn't have any interactive content, such as a link or button.
+
+We recommend giving users about 0.5 seconds per word before timing out a toast.
+
+## Composition Notes
+
+Keep toast text brief and scannable by placing only the highest priority information in it. Too much text in a toast can overwhelm a user. Aim for no more than 140 characters in a toast, and keep the text directly related to the user's current action. Success toasts generally run about 50 characters long, while all other alerts might run about 100 characters long. However, don't [sacrifice being direct and straightforward](/content/product-style-guide#be-direct-and-straightforward) for a shorter message.
+
+Be specific in your response to the user's action and if applicable, provide a resolution by using an [anchor](/components/anchor/). Be cautious about using a button in a toast since it may compete with other buttons on the screen and should be used only to trigger an action. If you must use a button, use a single [small secondary button](/components/button#small-button) in a toast.
```

Comment:
```suggestion
Be specific in your response to the user's action and if applicable, provide a resolution by navigating a user to another page using an [anchor](/components/anchor/). Be cautious about using a button in a toast since it should be used only to trigger an action. If you must use a button, use a [small secondary button](/components/button#small-button) in a toast.
```